codespell_lib/data/dictionary.txt

```diff
@@ -3816,6 +3870,7 @@ collegue->colleague
 collegues->colleagues
 collektion->collection
 colletion->collection
+collison->collision
```

Comment:
also "collusion"

Reply:
sorry, I can't. Reason: 'No collusion'  
Sorry, I couldn't help it :man_facepalming:  
59a316b